client/src/app/App.js

```diff
@@ -1197,12 +1197,27 @@ export class App extends PureComponent {
       this._closeNotification(id);
     };
 
+    // allow button to close notification
+    let contentWithButton;
+
+    if (content && content.type === 'button') {
+      contentWithButton = assign({},content);
+
+      const onClick = {
+        onClick:  ()=> {
+          content.props.onClick(); close();
+        }
+      };
+
+      contentWithButton.props = { ...content.props, ...onClick };
```

Comment:
Mh this looks like monkey patching to me. Is there a possibility to make this behavior part of the `Notification` component?